markdown/guide/creating-an-engine.md

```diff
@@ -179,6 +179,49 @@ Observant developers might also note that Addon's have an `app` directory in add
 
 ---
 
+### Generate in an in-repo Engine
```

Comment:
can you move it inside to `Create as In-Repo-Engine` on top file? makes more sense